docs/concepts/service-catalog/index.md

```diff
@@ -95,7 +95,7 @@ The following is a sequence diagram illustrating the steps involved in listing M
 
 1. Once the `ClusterServiceBroker` resource is added to Service Catalog, it triggers a *List Services* call to the external Service Broker.
```

Comment:
No italics for List Services. Maybe no caps. Not sure what List Services is. Is it a method name: ListServices? If so, then use backticks.

Reply:
I was trying to avoid the name of the actual API call in the Open Service Broker API by using their terminology. I have rewritten the three to avoid this awkwardness.